src/search.cpp

```diff
@@ -114,9 +147,22 @@ void BeamSearch_Cpu::SelectTop() {
   }
 
 #if 0
-  DumpMemory("Next Scores", next_scores);
-  DumpMemory("Next Tokens", next_tokens);
-  DumpMemory("Next Indices", next_indices);
+  // Print next_tokens, next_indices, next_scores
```

Comment:
replace it with the DumpSpan: https://github.com/microsoft/onnxruntime-genai/blob/00ceb80e1984c408459dfabe92a5b4eb97318578/src/models/debugging.h#L9